fiap-fintech/src/main/java/br/com/fiap/fintech/servlets/LogOutServlet.java

```diff
@@ -0,0 +1,23 @@
+package br.com.fiap.fintech.servlets;
```

Comment:
n tenho mt conhecimento sobre a forma que vc implementou login, to confiando em vc na implementação. próxima semana gostaria qeu vc me explicasse como implementou, se possível

Reply:
claro! o Filter é bem tranquilo, a FIAP passou a session em uma live tb, então podemos usar